package/kernel/linux/modules/multimedia.mk

```diff
@@ -0,0 +1,756 @@
+endef
+
+define KernelPackage/video-gspca-vc032x/description
+ The VC032X USB Camera Driver (vc032x) kernel module
+endef
+
+$(eval $(call KernelPackage,video-gspca-vc032x))
+
+
+define KernelPackage/video-gspca-zc3xx
+  TITLE:=zc3xx webcam support
+  KCONFIG:=CONFIG_USB_GSPCA_ZC3XX
+  FILES:=$(LINUX_DIR)/drivers/media/usb/gspca/gspca_zc3xx.ko
+  AUTOLOAD:=$(call AutoProbe,gspca_zc3xx)
+  $(call AddDepends/camera-gspca)
+endef
+
+define KernelPackage/video-gspca-zc3xx/description
+ The ZC3XX USB Camera Driver (zc3xx) kernel module
+endef
+
+$(eval $(call KernelPackage,video-gspca-zc3xx))
+
+
+define KernelPackage/video-gspca-m5602
+  TITLE:=m5602 webcam support
+  KCONFIG:=CONFIG_USB_M5602
+  FILES:=$(LINUX_DIR)/drivers/media/usb/gspca/m5602/gspca_m5602.ko
+  AUTOLOAD:=$(call AutoProbe,gspca_m5602)
+  $(call AddDepends/camera-gspca)
+endef
+
+define KernelPackage/video-gspca-m5602/description
+ The ALi USB m5602 Camera Driver (m5602) kernel module
+endef
+
+$(eval $(call KernelPackage,video-gspca-m5602))
+
+
+define KernelPackage/video-gspca-stv06xx
+  TITLE:=stv06xx webcam support
+  KCONFIG:=CONFIG_USB_STV06XX
+  FILES:=$(LINUX_DIR)/drivers/media/usb/gspca/stv06xx/gspca_stv06xx.ko
+  AUTOLOAD:=$(call AutoProbe,gspca_stv06xx)
+  $(call AddDepends/camera-gspca)
+endef
+
+define KernelPackage/video-gspca-stv06xx/description
+ The STV06XX USB Camera Driver (stv06xx) kernel module
+endef
+
+$(eval $(call KernelPackage,video-gspca-stv06xx))
+
+
+define KernelPackage/video-gspca-gl860
+  TITLE:=gl860 webcam support
+  KCONFIG:=CONFIG_USB_GL860
+  FILES:=$(LINUX_DIR)/drivers/media/usb/gspca/gl860/gspca_gl860.ko
+  AUTOLOAD:=$(call AutoProbe,gspca_gl860)
+  $(call AddDepends/camera-gspca)
+endef
+
+define KernelPackage/video-gspca-gl860/description
+ The GL860 USB Camera Driver (gl860) kernel module
+endef
+
+$(eval $(call KernelPackage,video-gspca-gl860))
+
+
+define KernelPackage/video-gspca-jeilinj
+  TITLE:=jeilinj webcam support
+  KCONFIG:=CONFIG_USB_GSPCA_JEILINJ
+  FILES:=$(LINUX_DIR)/drivers/media/usb/gspca/gspca_jeilinj.ko
+  AUTOLOAD:=$(call AutoProbe,gspca_jeilinj)
+  $(call AddDepends/camera-gspca)
+endef
+
+define KernelPackage/video-gspca-jeilinj/description
+ The JEILINJ USB Camera Driver (jeilinj) kernel module
+endef
+
+$(eval $(call KernelPackage,video-gspca-jeilinj))
+
+
+define KernelPackage/video-gspca-konica
+  TITLE:=konica webcam support
+  KCONFIG:=CONFIG_USB_GSPCA_KONICA
+  FILES:=$(LINUX_DIR)/drivers/media/usb/gspca/gspca_konica.ko
+  AUTOLOAD:=$(call AutoProbe,gspca_konica)
+  $(call AddDepends/camera-gspca)
+endef
+
+define KernelPackage/video-gspca-konica/description
+ The Konica USB Camera Driver (konica) kernel module
+endef
+
+$(eval $(call KernelPackage,video-gspca-konica))
+
+# DVB & tuners
+
+define KernelPackage/dvb-usb-v2
+  SUBMENU:=$(MMEDIA_MENU)
+  TITLE:=USB DVB support
+  KCONFIG:= \
+	CONFIG_MEDIA_DIGITAL_TV_SUPPORT=y \
+	CONFIG_MEDIA_ANALOG_TV_SUPPORT=y \
+	CONFIG_DVB_CORE=m \
```

Comment:
Could this be in a `kmod-dvb-core` package? Reason is that I have an Xbox One Digital TV Tuner which is a `CONFIG_DVB_USB` device (via dib0700). As that config tree has more dependencies (input & remote control – even through ironically my tuner has none) its probably better to not merge v1 and v2, but then the core should be separate.